pmptest/pom.xml

```diff
@@ -1,10 +1,10 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.mosip.test</groupId>
-  <artifactId>pmpui</artifactId>
+  <artifactId>uitest-pmp</artifactId>
   <version>1.2.1</version>
-  <name>pmp-ui-automation</name>
-  <description>pmp-ui-automation</description>
+  <name>uitest-pmp</name>
+  <description>uitest-pmp</description>
```

Comment:
Update the description as "Parent project of MOSIP Partner Management Portal - UI Tests"

Reply:
updated